bin/MessagePack.SourceGenerator/Debug/netstandard2.0/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "com.github.messagepack.sourcegenerator.internal",
+    "displayName": "MessagePack Source Generator Internal",
+    "author": {
+        "name": "MessagePack-CSharp",
+        "url": "https://github.com/MessagePack-CSharp/MessagePack-CSharp"
+    },
+    "version": "1.0.0",
```

Comment:
This is necessary for referencing pre-built DLLs from the Unity side for development purposes.
`MessagePack/Debug/netstandard2.1` is also processed similarly.
The version has no significance; it's purely for reference during development and is not used at all during release.
We're using Debug instead of Release because typically, during normal development, Debug is where most changes occur.

Reply:
I think that answer would be great to put into a README in this directory, since otherwise I would think this was a mistake and delete them from git.

Reply:
I added a note to the ReadMe in UnityClient and a guide to it from CONTRIBUTING.md.